CWE/common/countries/Tanzania.txt

```diff
@@ -12,59 +12,135 @@ party = {
 		end_date = 2100.1.1
 }
 party = {
-		name = "party_big_tent"
+		name = "African National Congress"
```

Comment:
For instances where there are multiple parties of the same name, I think its best to add which faction or ideology they represent. So, for example "African National Congress (Centrist)" or "African National Congress ([Faction name])". It just helps with being able to both point to the party in files and seeing which party is in charge at a glance.

Reply:
Alright